wsd/protocol.txt

```diff
@@ -242,9 +242,12 @@ dialog <command>
 
     <command> is unique identifier for the dialog that needs to be painted.
 
-uno <command>
+uno <command> [serial=text] [arguments]
```

Comment:
Is this 'serial=' correct? I don't see any output from 'git grep serial=', but perhaps I miss something.